src/theme.ts

```diff
@@ -256,6 +256,32 @@ export const theme = createTheme({
 				},
 			},
 		},
+		MuiDialogTitle: {
+			styleOverrides: {
+				root: {
+					padding: '24px',
+					fontSize: '24px',
+					fontFamily: '"Source Sans Pro", Helvetica, sans-serif',
```

Comment:
This should not be necessary, as all strings should live inside a Typography component. Having said that if really needed it should be possible to make a generic rule, this might be of help https://mui.com/material-ui/react-css-baseline/

Reply:
@JSReds I agree that the font-family rule should be generic but we can keep it like this for now, I'll add the css baseline (probably scoped as not to interfere too much with Grommet) in a separate PR.